librz/core/cbin.c

```diff
@@ -2276,7 +2276,7 @@ static int bin_symbols(RzCore *r, PJ *pj, int mode, ut64 laddr, int va, ut64 at,
 		}
 	}
 	if (IS_MODE_NORMAL(mode)) {
-		rz_table_set_columnsf(table, "dssssdss", "nth", "paddr", "vaddr", "bind", "type", "size", "lib", "name");
+		rz_table_set_columnsf(table, "dXXssdss", "nth", "paddr", "vaddr", "bind", "type", "size", "lib", "name");
```

Comment:
so before paddr and vaddr were considered as strings??

Reply:
Yep, these fields were converted directly with `sdb_fmt()`.